### DISCUSSION TITLE
Units (mi,$,etc

### DISCUSSION THREAD

**Discussion by prussell69** [CarlosCuezva/dashboards-Grafana-Teslamate#5] [General]
Love the dashboards. Since I'm not very savy at this, is there an easy way to show miles instead of kilometers, dollars instead of euros, etc.?

**Comment by CarlosCuezva** [comment]
I'm glad you like it, if you give me a few days I'll adapt the dashboards to support km/mi or €/$

**Reply by prussell69** [reply]
Thanks. The layout and information looks great. 
I'm very impressed.

**Comment by CarlosCuezva** [comment]
Hello!!

I just published a new version with the changes you have indicated, I hope they help you.

I've removed the currency symbol because Teslamate doesn't allow you to select your currency; but there is support for miles/kilometers, now the data is converted automatically.

**Comment by prussell69** [comment]
Thank you. Looks great. I am having an issue on the 'current-charge-view' dashboard. I have several panels that are only showing 'No Data'. Similar panels (Voltage) display on the overview, but not this dashboard. Not sure why.
All the other dashboards seem to be OK.
![image](https://user-images.githubusercontent.com/28122849/212549593-7d40e675-a29d-4fb5-840b-7502eaae381e.png)

**Reply by CarlosCuezva** [reply]
Can you mouse over the error symbol of any of them and show me the error message?

e.g.

![Captura de pantalla 2023-01-15 a las 16 49 34](https://user-images.githubusercontent.com/5815421/212551255-1671e4e6-a60c-483d-919b-ed140dab2825.png)

**Comment by prussell69** [comment]
This is the Voltage panel:
![image](https://user-images.githubusercontent.com/28122849/212552244-431715a7-be0c-4ae2-85ff-4d0e79ab42c9.png)

**Comment by prussell69** [comment]
BTW, these were showing during a charge also.

**Comment by CarlosCuezva** [comment]
Sorry, I hadn't noticed the message.

Can you check the value that the variable "charging_processes" gives you?

![image](https://user-images.githubusercontent.com/5815421/213167455-81224da6-b7f1-49d8-ae1a-5e7ab858a237.png)

**Reply by prussell69** [reply]
"None"
I found If I linked it from my 'Charges' list (Specific charge) It will fill with data. I used this for the link:
d/#########?from=${__data.fields.start_date_ts.numeric}&to=${__data.fields.end_date_ts.numeric}&var-car_id=${__data.fields.car_id.numeric}&var-charging_process_id=${__data.fields.id.numeric}

Only the latest charge would fill the data completely, the others are missing the 'elapsed time', 'voltage', and 'amps' fields.
Is there a better way to call this dashboard? Did you have a link already I just missed it?

**Reply by CarlosCuezva** [reply]
That query can't return "_None_", because that means you haven't started any recharge process.

**Comment by prussell69** [comment]
I found if I add this to end of the url, it comes up OK:
&var-car_id=1&var-charging_process_id=210&orgId=1
Once I do that, the variable now shows '211'
Hope that helps a little bit.

**Reply by CarlosCuezva** [reply]
Okkk, I see that you are using an older version of the dashboard.

The "charging_process_id" variable has been removed, try using the new version of that dashboard.

**Comment by prussell69** [comment]
Your 'current-charge-view' dashboard, or something different?

**Comment by CarlosCuezva** [comment]
Yes, that dashboard
